src/components/SearchingMap.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { withLeaflet, MapControl } from "react-leaflet";
+import { OpenStreetMapProvider, GeoSearchControl } from "leaflet-geosearch";
+
+class GeoSearch extends MapControl {
+    constructor(props, context) {
+        super(props);
+    }
+
+    createLeafletElement(opts) {
+        const provider = new OpenStreetMapProvider();
```

Comment:
From what I see we can use `customProvider`
![Screenshot 2020-04-08 at 18 05 09](https://user-images.githubusercontent.com/6861120/78806949-ad390400-79c3-11ea-9625-6de8e3be9ad7.png)